Automattic/wp-calypso
# Stats: Release the new 7-day highlights feature

#### Proposed Changes

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/4044428/201223613-a1b0088f-f6c3-4279-9275-777857dafc03.png">

* Releases the new 7-day highlights section for the Traffic Stats page.
* Adds trend arrows to each highlight card.

#### Testing Instructions



* Spin up this change in a live branch.
* Navigate to Stats.
* Ensure you see the new 7-day highlights section.

#### Pre-merge Checklist






Related to #69224.
